projects/aca-playwright-shared/src/page-objects/components/search/search-input.component.ts

```diff
@@ -44,7 +44,7 @@ export class SearchInputComponent extends BaseComponent {
   }
 
   async performDoubleClickFolderOrFileToOpen(name: string): Promise<void> {
-    await this.getCellLinkByName(name).waitFor({ state: 'visible', timeout: timeouts.normal });
+    await this.getCellLinkByName(name).waitFor({ state: 'visible', timeout: timeouts.medium });
```

Comment:
Are we 100% sure that we want to increase the timeout here?

Reply:
Yes here, I check for 2-3 runs. test get flaky with less timeout. specially for Share file scenarios.